ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md

```diff
@@ -0,0 +1,258 @@
+## Arctica Greeter - Installation Guide for Ubuntu 16.04 LTS (Latest version)
+
+# Introduction
+
+The following document, documents how to install Ubuntu with Arctica Greeter. Make sure when typing the commands for installation, since Ubuntu differ between uppercase and lowercase letters because everything is case-sensitive. Everything that is highlighted with a black background is what you would type in the Terminal.
+
+## Stage one
+ 
+**Install Ubuntu:**
+When installing Ubuntu make sure to check both boxes, **Download Updates While Installing Ubuntu** and **Installation of 3rd party applications**, when they both are ticked then press continue. </span>
+![1](https://github.com/Korn20000/Testing/blob/master/img/1.png?raw=true)
+
+Next step choose **Erase Disk** and press Install Now.
+If it asks you **“Write the changes to disk?”**, ignore it and press continue.
+
+![2](https://github.com/Korn20000/Testing/blob/master/img/2.png?raw=true)
+
+Next choose location where you live, after the location has been chosen you will get the opportunity to choose your keyboard layout so it can fit to your language.
+
+At the end it will ask you to fill in a form with name, computer name, username and password. Make sure that you do not login automatically since it will give complications with login on Arctica Greeter.
+
+![3](https://github.com/Korn20000/Testing/blob/master/img/3.png?raw=true)
+
+Restart, when the installation is done.
+
+## Stage two
+
+After Ubuntu is successfully installed open a terminal either by pressing **CTRL+ALT+T** or navigate to **“Search your computer”** and search for terminal.
+
+![4](https://github.com/Korn20000/Testing/blob/master/img/4.png?raw=true)
+
+
+When the **“Terminal”** appears you will see a window similar to the picture below.
+
+![5](https://github.com/Korn20000/Testing/blob/master/img/5.png?raw=true)
+
+Now that the **“Terminal”** is open, run the following commands:
+
+- **sudo -i** 

```

Comment:
for command line instructions, better use three backticks (as block start marker) and another three backticks (as block end markers). The layout engine will then create a code box around the text.

See this example:

```
apt-get install markdown-editor
```